### PR TITLE
>add meme arrows ( ͡° ͜ʖ ͡°)

### PR DESCRIPTION
**Pull request by SolarisFox**
I don't understand how we've gone so long without the best formatting option.

### CODE REVIEW

**Comment by Morfent** [comment]
Who are you quoting?

**Comment by Ridaz** [comment]
Please, not this again

**Comment by kotarou3** [comment]
:+1:

**Comment by Joimer** [comment]
This pull request is wrong and has a missleading name.
Please rename it to "Implement le acute angle may may"

**Comment by Zarel** [comment]
I added meme arrows two years ago, they were massively abused and I took them out a few hours later.

**Comment by DanUgore** [comment]
@SolarisFox Stop being so new :^)

File: js/battledata.js [hunk 1]
```diff
@@ -358,6 +358,10 @@ var Tools = {
 
 		var options = Tools.prefs('chatformatting') || {};
 
+		// >greentext
+		if (!options.hidegreentext && str.substr(0, 4) === '&gt;' && str.substr(4, 8) !== '&gt;') {
+			str = '<font color=#789922>' + str + '</font>';
```
Comment (Zarel) [inline js/battledata.js]: \>`<font>`

\>2015

**Pull request closed without merging**